unity-client/Assets/Scripts/MainScripts/DCL/Components/TextShape/TextShape.cs

```diff
@@ -86,7 +88,7 @@ public static IEnumerator ApplyModelChanges(ParcelScene scene, TMP_Text text, Mo
 
             text.text = model.value;
 
-            text.color = new Color(model.color.r, model.color.g, model.color.b, model.color.a);
+            text.color = new Color(model.color.r, model.color.g, model.color.b, model.visible ? model.opacity : 0);
```

Comment:
Why are we using `model.opacity` now instead of `model.color.a`?

Reply:
In `kernel` we are combining `Color3 color` and `float opacity` but Unity has no implementation of Color3, all of them has alpha. So basically color.a does nothing because the value is defaulted.

We have to ways to solve this:
1) Remove opacity and use a Color4
2) Maintain opacity and use it in Unity instead of opacity.

After talking with Nico we decided that the second one was less disruptive for the creators.